index.yaml

```diff
@@ -5077,6 +5077,36 @@ entries:
     - assets/rancher-external-ip-webhook/rancher-external-ip-webhook-0.1.400.tgz
     version: 0.1.400
   rancher-gatekeeper:
+  - annotations:
+      catalog.cattle.io/auto-install: rancher-gatekeeper-crd=match
+      catalog.cattle.io/certified: rancher
+      catalog.cattle.io/display-name: OPA Gatekeeper
+      catalog.cattle.io/kube-version: '>= 1.20.0-0 < 1.25.0-0'
```

Comment:
Does this version of gatekeeper support 1.25? I believe we intend to support k8s 1.25 in Rancher 2.7.1, so if this chart supports it we should raise this.

Reply:
done